src/StorageAdapter.ts

```diff
@@ -13,6 +13,7 @@ export class StorageAdapter {
   writeInStorage<T>(name: string, content: T): Promise<void | Error> {
     return new Promise((resolve, reject) => {
       const contentString = JSON.stringify(content);
```

Comment:
@quarrant 

I think we should add a `jsonify` flag like [mobx-persist](https://github.com/pinqy520/mobx-persist#createconfig)? This way if I am using localForage I can pass it the raw object and it can handle it.

Reply:
Yes, i think it's a good idea